en/docs/basics_topics/model_load_save.md

```diff
@@ -240,42 +237,34 @@ lenet_models_name
 
 We can see:
 
-* The job function in network, each of the variable have a sub-directory.
+* The network in job function, each variable is corresponding to a sub-directory.
 
-* All subdirectories above have an  `out` document. It store the parameters of network in binary. `Out` is the default file name. We can change that by  `variable op` in the network.
+* All subdirectories above have an  `out` document. It store the parameters of network in binary form. `Out` is the default file name. We can change that by  `variable op` in the network.
 
-* `Snapshot_done` is an empty folder. If it exists, means the network is completed the training.
+* `Snapshot_done` is an empty folder. If it exists, it means that the network has been trained. 
```

Comment:
it means that the network has been trained ->it means that the network training has been finished